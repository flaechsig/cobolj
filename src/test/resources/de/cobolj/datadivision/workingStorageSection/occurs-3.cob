IDENTIFICATION DIVISION.
PROGRAM-ID. OCCURS-TEST.

DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-ARRAY.
     05 WS-HELLO PIC A(10) VALUE 'HELLO'.
     05 WS-TEST PIC A(5) OCCURS 3.
     05 WS-END PIC A(5) VALUE 'ENDE'.

PROCEDURE DIVISION.
MOVE 'A' to WS-TEST(1).
MOVE 'B' TO WS-TEST(2).
MOVE 'C' TO WS-TEST(3).
DISPLAY WS-ARRAY '#'.
STOP RUN.