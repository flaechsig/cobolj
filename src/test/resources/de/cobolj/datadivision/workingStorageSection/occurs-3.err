Füllen des Arrays und Ausgeben (occurs ist eingebettet)