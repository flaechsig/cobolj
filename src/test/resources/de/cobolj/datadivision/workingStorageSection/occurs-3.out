HELLO     A    B    C    ENDE #
